docker/app_img/Dockerfile

```diff
@@ -1,72 +1,60 @@
+# NOTE: due to newer version of mkfs.ext4 will enable some flags
+#       by default that are not supported by older version of ubuntu,
+#       we have to pin the ubuntu base to 22.04.
+# NOTE: due to glibc compat issue, we need to pair ubuntu:22.04 with bullseye.
 ARG UBUNTU_BASE=ubuntu:22.04
+ARG BUILD_BASE=python:3.13-bullseye
```

Comment:
The reason for selecting python3.13 is its stability, correct?
Do we test whether otaclient works with python3.13? If not, let’s add a test Docker image 👍

Reply:
Thank you, you're right.
How about adding the following as well?
- Update pyproject.toml (add Programming Language :: Python :: 3.13)
- Update sonar-project.properties
- (.pre-commit-config.yaml)
- Add a Python 3.13 test image (for example, Ubuntu 22.04 + Python 3.13)